lib/widgets/theme.dart

```diff
@@ -154,6 +154,7 @@ class DesignVariables extends ThemeExtension<DesignVariables> {
     composeBoxBg: const Color(0xffffffff),
     contextMenuCancelText: const Color(0xff222222),
     contextMenuItemBg: const Color(0xff6159e1),
+    contextMenuItemIcon: const Color(0xff4f42c9),
     contextMenuItemLabel: const Color(0xff242631),
     contextMenuItemMeta: const Color(0xff626573),
     contextMenuItemText: const Color(0xff381da7),
```

Comment:
button: Fix MenuButton's icon color
    
    Oops, this wasn't following the Figma. Fixed.

Hmm good catch.

I guess this affects the icons in the action sheets? Is that the existing place this affects?

Looks like the difference is pretty subtle, thankfully. Here's a quick preview, from my IDE showing color swatches in the left gutter next to the source code:
![image](https://github.com/user-attachments/assets/d8f1fde3-d23b-4d9e-bfcd-826883e07c83)

Bringing those right next to each other (new above, old below):
![image](https://github.com/user-attachments/assets/7be80f26-e8e1-4dc2-8eab-c21057ee00e3)

Reply:
Yep yep, and I posted a screenshot of the change in the action sheet: https://github.com/zulip/zulip-flutter/pull/1631#issuecomment-3051130010